test/Microsoft.Extensions.Localization.Tests/LocalizationServiceCollectionExtensionsTest.cs

```diff
@@ -21,15 +21,23 @@ public void AddLocalization_AddsNeededServices()
 
             // Assert
             var services = collection.ToList();
-            Assert.Equal(2, services.Count);
+            Assert.Equal(4, services.Count);
 
-            Assert.Equal(typeof(IStringLocalizerFactory), services[0].ServiceType);
-            Assert.Equal(typeof(ResourceManagerStringLocalizerFactory), services[0].ImplementationType);
+            Assert.Equal(typeof(IOptions<>), services[0].ServiceType);
```

Comment:
Is Options a required service for Localization though? If it is required to be there, we should make sure it's added when `AddLocalization` is called no? @javiercn?